ui/src/components/MENU.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import { Link as LinkRouter } from "react-router-dom";
+import { Typography, Menu } from "antd";
+import "./AT_HEADER.css";
+import "antd/dist/antd.css";
```

Comment:
you don't need to import `antd.css` in all files :) could you please just leave it in the `index.js`?